File: app-ios/tutanota/Sources/Utils/CancelledError.swift
import TutanotaSharedFramework

class CancelledError: TutanotaError {
	init(message: String, underlyingError: Error) { super.init(message: message, underlyingError: underlyingError) }

	override var name: String { get { "de.tutao.tutanota.CancelledError" } }
}

File: src/common/api/common/error/CancelledError.ts
//@bundleInto:common-min

import { TutanotaError } from "@tutao/tutanota-error"

export class CancelledError extends TutanotaError {
	/**
	 * A cancelled error is thrown when a async action is aborted
	 * @param message An information about the exception.
	 * @param error The original error that was thrown.
	 */
	constructor(message: string) {
		super("CancelledError", message)
	}
}
